docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1098,6 +1099,135 @@ team can more easilly read the messages.
+
+```
+	.../fhir-bulkimexport-connectathon/6xjd4M8afi6Xo95eYv7zPxBqSCoOEFywZLoqH1QBtbw=/Patient_1.ndjson
+```
+Following is the beautified response of sample polling location request after the export is finished:
+
+```
+{
+"transactionTime": "2020/01/20 16:53:41.160 -0500",
+"request": "/$export?_type=",
+"requiresAccessToken": false,
+"output" : [
+  { "type" : "AllergyIntolerance", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/AllergyIntolerance_1.ndjson", 
+    "count": 20},
+  { "type" : "AllergyIntolerance", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/AllergyIntolerance_2.ndjson", 
+    "count": 8},
+  { "type" : "Observation", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Observation_1.ndjson", 
+    "count": 234},
+  { "type" : "Observation", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Observation_2.ndjson", 
+    "count": 81},
+  { "type" : "Group", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Group_1.ndjson", 
+    "count": 15},
+  { "type" : "Condition", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Condition_1.ndjson", 
+    "count": 30},
+  { "type" : "Condition", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Condition_2.ndjson", 
+    "count": 21},
+  { "type" : "Composition", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Composition_1.ndjson", 
+    "count": 20},
+  { "type" : "Composition", 
+      "url": "https://s3.us-south.cloud-object-storage.appdomain.cloud/fhir-bulkimexport-connectathon/6SfXzbGvYl1nTjGbf5qeqJDFNjyljiGdKxXEJb4yJn8=/Composition_2.ndjson", 
+    "count": 8}]
+}
+
+```
+
+The exported ndjson file is configured with public access automatically and with 2 hours expiration time, the randomly generated secret in the path is used to protect the file. please notice that IBM COS doesn't support expiration time for each single COS object, so please configure retention policy (e.g, 1 day) for the bucket if IBM COS is used. And for both Amazon S3 and IBM COS, please remember that public access should never be configured to the bucket itself.  
```

Comment:
I am not sure I follow this?  

If a file is written out to Object Storage, it's public by default? only protected by obfuscation? 

The above paragraph is a bit tricky to read

Reply:
hum, this doesn't seem ideal.  Are there any alternatives?

Reply:
The other similar choice is the pre-signed urls which provide temp url for read and write, but the idea is the similar - a temp url with secret(signature) in path and can expire automatically.  https://cloud.ibm.com/docs/services/cloud-object-storage?topic=cloud-object-storage-presign-url.    and this approach is a little bit complicated than what I did in codes.   This is the url format of this approach:     
    var requestUrl = endpoint + '/' +
    bucket + '/' +
    objectKey + '?' +
    standardizedQuerystring +
    '&X-Amz-Signature=' +
    signature;

Reply:
I'm curious what Lee thinks.  Otherwise looks good.

Reply:
Albert and I came up with this approach together.  If there is enough "entropy" in the name, I think it would be acceptable for at least some use cases.  Similar to a box "share" link or a password reset link you get via email.  I do wish we could totally hide the internals though (i.e. not leak our bucket name).
We should be very clear that its publicly accessible though.

Reply:
Yeah - I 100% agree.

Reply:
One design goal for this was to keep us out of playing "middleman" for these downloads.  In most of our deployments we have a pretty short http timeout (60 seconds), so the download sort of needs to be a direct connection with COS.  
http://hl7.org/fhir/uv/bulkdata/STU1/authorization/index.html explains how to use SMART authentication, but I'm not sure how to do that.

Reply:
I thought about generate temp bear token for the download, but the problem is that the bear token could give the user full access to the bucket even COS service. maybe we could create a spec user/apikey without any access by default, and then give read access in the object ACL to this user, and generate the bear token for user to download the object, this will be much more complicated ...